src/AzureClient/Mocks/MockQIRSubmitter.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Quantum.Runtime;
+using Microsoft.Quantum.Runtime.Submitters;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Quantum.IQSharp.AzureClient
+{
+    internal class MockQIRSubmitter : IQirSubmitter
+    {
+        public string Target => throw new NotImplementedException();
+
+        private IReadOnlyList<Argument> ExpectedArguments = new List<Argument>();
```

Comment:
Could you please add a documentation comment on the `ExpectedArguments` member explaining its purpose?

Reply:
done